tests/test_distributions.py

```diff
@@ -302,6 +305,18 @@ def test_expand_margins(self, flat_array):
         assert x1 == x2
         assert y1 + height * 2 == pytest.approx(y2)
 
+    def test_multiple_rugs(self, long_df):
+
+        values = np.linspace(start=0, stop=1, num=5)
+        ax = lineplot(x=values, y=values)
+        rugplot(x=values, ax=ax)
+        ylim = ax.get_ylim()
+
+        for j in range(4):
+            rugplot(x=values, ax=ax, expand_margins=False)
+
+        assert_array_equal(ylim, ax.get_ylim())
```

Comment:
```suggestion
        assert ylim == ax.get_ylim()
```
Nit: these will both be tuples, so simple equality comparison should be sufficient.

Reply:
TIL. I've replaced it with your suggestion 👍